msgpack/index.d.ts

```diff
@@ -81,3 +81,7 @@ interface MsgPackCallbackResult {
 }
 
 declare var msgpack: MsgPackStatic;
+
+declare module "msgpack" {
```

Comment:
The appropriate fix is to replace this with

```ts
export = msgpack;
```

If `msgpack` is also available in as a global in the browser (it doesn't look like it is, but I'm not sure), then it'd be

```ts
export = msgpack;
export as namespace
```